tests/system/src/test/java/test/renderlock/RenderLockCommon.java

```diff
@@ -108,19 +115,13 @@ public static void doSetupOnce() throws Exception {
         // focusLost event with the lock held
         assumeTrue(PlatformUtil.isMac() || PlatformUtil.isWindows());
 
-        // Start the Application
-        new Thread(() -> Application.launch(MyApp.class, (String[])null)).start();
-
-        if (!launchLatch.await(TIMEOUT, TimeUnit.MILLISECONDS)) {
-            fail("Timeout waiting for Application to launch");
-        }
-
+        Util.launch(launchLatch, MyApp.class);
         assertEquals(0, launchLatch.getCount());
     }
 
     @AfterClass
     public static void doTeardownOnce() {
-        Platform.exit();
+        Util.shutdown();
```

Comment:
This change causes the test to fail on Linux due to a latent test bug (specific to this test). The test is skipped on Linux using `assumeTrue` in an `@BeforeClass` method. This causes all test methods to be skipped, but the `@AfterClass` method is still run unconditionally. The `Util.shutdown` method calls `Platform.runLater` which will throw an exception if the FX runtime was never started. This causes JUnit to fail with a "DefaultMultiCauseException".

I recommend either moving the `assumeTrue` to the `@Test` method in the `RenderLock1Test` class (there is only one) or else qualifying the call to `Util.shutdown` with `if (myApp != null)` so we don't call it if the platform wasn't launched.